.devcontainer/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/devcontainers/base:ubuntu
+
+RUN wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor | tee /usr/share/keyrings/hashicorp-archive-keyring.gpg
+RUN echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/hashicorp.list
+
+# Install Terraform
+RUN apt-get update && apt-get install -y \
+    wget \
+    unzip \
```

Comment:
(non-blocking) Since `wget` is used earlier you probably don't need it here. I suspect unzip may be part of the base ubuntu image as well.

Reply:
Yes you're right good catch! Removed